content/pt-br/docs/tasks/configure-pod-container/extended-resource.md

```diff
@@ -0,0 +1,139 @@
+---
+title: Atribuindo Recursos Estendidos a um Contêiner
+content_type: task
+weight: 40
+---
+
+<!-- overview -->
+
+{{< feature-state state="stable" >}}
+
+Esta página mostra como atribuir recursos estendidos a um Contêiner.
+
+
+## {{% heading "prerequisites" %}}
+
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+Antes de fazer este exercício, faça o exercício em
+[Anunciar recursos estendidos para um Nó](/docs/tasks/administer-cluster/extended-resource-node/).
+Isso configurará um de seus nós para anunciar um recurso de `dongle`.
+
+<!-- steps -->
+
+## Atribua um recurso estendido a um Pod
+
+Para solicitar um recurso estendido, inclua o campo `resources:requests` no seu 
+manifesto do contêiner. Recursos estendidos são totalmente qualificados 
+com qualquer domínio fora do `*.kubernetes.io/`. Nomes de recursos estendidos válidos 
+tem a forma de `example.com/foo`, onde `example.com` é substituído pelo domínio 
+da sua organização e `foo` é um nome descritivo de recurso.
+
+Aqui está o arquivo de configuração para um pod que possui um contêiner:
+
+{{< codenew file="pods/resource/extended-resource-pod.yaml" >}}
+
+No arquivo de configuração, você pode ver que o contêiner solicita 3 `dongles`.
```

Comment:
Same suggestion here. It makes more sense to use italic as it is a term from a different language, rather than a coding-specific term.